doc/ven_impl/ven_template/report.md

```diff
@@ -0,0 +1,30 @@
+# CNTT - VI Report
+
+<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
+
+## Table of Contents
+* [1 Introduction](#1)
+* [2 Test Suite](#2)
+* [3 Step by Step instructions](#3)
+* [4 Results](#4)
+
+<a name="1"></a>
+## 1 Introduction
+
+
+<a name="2"></a>
+## 2 Test Suite
+> Beyond RC, more details about test suite executed.
+
+<a name="3"></a>
+## 3 Step by Step instructions
+
+> Step by step instructions to run RCx against the implementation.
+> Should adopt similar process as written in RC-1
+
+<a name="4"></a>
+## 4 Results
+
+> Detailed results of the test outputs.
+> if possible, pointers and links to public results (such as those of OVP)
+> Links to LFN badges, etc.
```

Comment:
Sorry if I misunderstood, the approvals are in the Vendor Implementation report (has links to these Test Results) so how will there e badges in the Test Report? Also, are there badges during the "trials"?

Reply:
same template will be used for trials or for simply a vendor to provide their own conformant implementations. in either case, those has to be signed off by CNTT members. the way we sign them off can be by relying on the badge as well as someone to validate the instructions and nothing is broken.